src/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Gregorio configuration headers. Copyright (C) 2015 Elie Roux
```

Comment:
`Gregorio project` or `The Gregorio project` or `Gregorio project authors`?  gregorio-utils.c has `Gregorio project` in the top-comment and `Gregorio project authors` in `print_licence()`.

If you care about my vote, I like `The Gregorio Project` (using title case), but I'm no legal expert or anything like that.